static/scripts/index.js

```diff
@@ -0,0 +1,513 @@
+                                if ((load_q1 === 1 && load_q2 === 0) || (load_q1 === 0 && load_q2 === 1)) {
+                                    TD.style.backgroundColor = 'green';
+                                }
+                            }
+                        }
+                    }
+                }
+            },
+            customBorders: [
+                {
+                    range: {
+                        from: {
+                            row: 0,
+                            col: 0,
+                        },
+                        to: {
+                            row: nbrLines,
+                            col: 6,
+                        },
+                    },
+                    end: {
+                        width: 2,
+                        color: 'black'
+                    },
+                },
+                {
+                    range: {
+                        from: {
+                            row: 0,
+                            col: 0,
+                        },
+                        to: {
+                            row: 5,
+                            col: nbrCols,
+                        },
+                    },
+                    bottom: {
+                        width: 2,
+                        color: 'black'
+                    },
+                },
+                {
+                    row: 5,
+                    col: 6,
+                    bottom: {
+                        width: 2,
+                        color: 'black'
+                    }
+                },
+            ],
+            beforeFilter(conditionsStack) {
+                const filtersPlugin = this.getPlugin('filters');
+                const tab = this.getData();
+
+                let values = [];
+                const filteredResults = [];
+                const col = conditionsStack[0].column;
+
+                if (conditionsStack && conditionsStack.length > 0) {
+                    for (let i = 0; i < conditionsStack.length; i++) {
+                        values = conditionsStack[i].conditions[0].args.flat();
+
+                        for (const row of tab) {
+                            if (values.includes(row[col])) {
+                                filteredResults.push(row[0]);
+                            }
+                        }
+                    }
+                }
+                filtersPlugin.clearConditions();
+                filtersPlugin.addCondition(0, 'by_value', [filteredResults]);
+            },
+            afterFilter(conditionsStack) {
+                const filtersPlugin = this.getPlugin('filters');
+                const filtersRowsMap = filtersPlugin.filtersRowsMap;
+
+                // Exclude fixed lines from the filter
+                for (let i = 0; i < lenFixedRowsText; i++) {
+                    filtersRowsMap.setValueAtIndex(i, false);
+                }
+            },
+            licenseKey: "non-commercial-and-evaluation",
```

Comment:
Do we really need to specify this ? :eyes:

Reply:
If I remove this property, this message appears below the table: 

‘The licence key for Handsontable is missing. Use the key you purchased to activate the product. Alternatively, you can activate Handsontable for non-commercial use by passing the key: ‘non-commercial-and-evaluation’. For more information, consult the documentation or contact us at support@handsontable.com. ’